src/promptflow-evals/README.md

```diff
@@ -0,0 +1,117 @@
+# Prompt flow evaluators
+
+[![Python package](https://img.shields.io/pypi/v/promptflow-evals)](https://pypi.org/project/promptflow-evals/)
+[![License: MIT](https://img.shields.io/github/license/microsoft/promptflow)](https://github.com/microsoft/promptflow/blob/main/LICENSE)
+
+## Introduction
+Evaluators are prebuilt promptflow pipelines that are designed to measure the quality of the outputs from language models.

```

Comment:
These can be custom in addition to prebuilt

Reply:
Fixed.